docs/5-zarf-tutorials/8-custom-init-packages.md

```diff
@@ -83,9 +83,22 @@ $ zarf package create . --set AGENT_IMAGE_TAG=vX.X.X \
 
 :::
 
+:::note
```

Comment:
I can't make this as a suggestion in Github because of the lines but I think this would be better wording if we folded this into the above `:::tip`:

```
:::tip

If your enterprise uses pull-through mirrors to host vetted images you can run the following command to create a Zarf 'init' package from those mirrors (where `<registry>.enterprise.corp` are your enterprise mirror(s)):

```
$ zarf package create . \
  --set AGENT_IMAGE_TAG=$(zarf version) \
  --registry-override docker.io=dockerio.enterprise.corp \
  --registry-override ghcr.io=ghcr.enterprise.corp \
  --registry-override quay.io=quay.enterprise.corp
```

And if you need even more control over the exact Agent, Registry, and Gitea images you can specify that with additional `--set` flags:

```
$ zarf package create . \
  --set AGENT_IMAGE_TAG=$(zarf version) \
  --set AGENT_IMAGE="opensource/zarf" \
  --set AGENT_IMAGE_DOMAIN="custom.enterprise.corp" \
  --set REGISTRY_IMAGE_TAG=2.8.3 \
  --set REGISTRY_IMAGE="opensource/registry" \
  --set REGISTRY_IMAGE_DOMAIN="custom.enterprise.corp" \
  --set GITEA_IMAGE="custom.enterprise.corp/opensource/gitea" \
  --set GITEA_SERVER_VERSION="v1.19.3"
```

⚠️ - The Gitea image and version are different than the Agent and Registry in that Zarf will always prefer the `rootless` version of a given server image.  This means that the above reference would template out to be `custom.enterprise.corp/opensource/gitea:v1.19.3-rootless`.  If you need to change this, edit the `packages/gitea` package.

You can find all of the `--set` configurations by looking at the `zarf-config.toml` in the root of the repository.

:::
```

Reply:
(the backticks are formatted strange but that should be the gist.)

Reply:
I also have tried to leave specific references to `registry1.dso.mil` out of our docs to keep things more generic (you will only find that URL in the Big Bang tutorial)